packages/react-native-select-pro/src/components/select-field-type/select-field-type.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import type { TextStyle, ViewStyle } from 'react-native';
+import { StyleSheet, Text, View } from 'react-native';
+
+import { COLORS, FONT_SIZE, PADDING } from '../../constants/styles';
+import type { OptionType } from '../../index';
+import { MultiSelect } from '../multi-select';
+import { SelectInput } from '../select-input';
+
+import type { SelectFieldTypeProps } from './select-field-type.types';
+
+export const SelectFieldType = ({
+    isOpened,
+    selectedOption,
+    onPressSelectControl,
+    dispatch,
+    disabled,
+    multiSelection,
+    placeholderText,
+    placeholderTextColor,
+    searchable,
+    searchPattern,
+    textInputProps,
+    searchValue,
+    setPosition,
+    onPressRemove,
+    containerStyle,
+    textStyle,
+    multiSelectionOptionStyle,
+}: SelectFieldTypeProps) => {
+    const multiSelect = (
+        <MultiSelect
+            disabled={disabled}
```

Comment:
and we can try to use in v2 this convention:
`{...{ dispatch, isOpened etc. }`

so you can implementing it now